src/ralph/attachments/models.py

```diff
@@ -41,6 +28,15 @@ def get_attachments_for_object(self, obj):
             items__object_id=obj.id,
         )
 
+    def create_from_file_path(self, file_path, uploaded_by):
+        attachment = self.model()
+        attachment.uploaded_by = uploaded_by
+        with open(file_path, 'rb') as f:
+            content = ContentFile(f.read())
+            attachment.file.save('test', content, save=True)
```

Comment:
it's name of file it can be anything cause after save ``get_file_path`` override it